actionkit_templates/templatetags/actionkit_tags.py

```diff
@@ -272,6 +272,22 @@ def multiply(value, arg):
 def percent_of(value, arg):
     return '%.1f' % (100 * (float(value) / float(arg)))
 
+@register.filter
+def mask_end(value, arg):
+    # given a string and a number, it shows the first X characters of string, ending with "..."
+    short_string = value[:arg]
+    ellipses = "..."
+    with_ellipses = short_string + ellipses
+    return with_ellipses
+
+@register.filter
+def mask_beginning(value, arg):
```

Comment:
I don't see this documented here:
https://roboticdogs.actionkit.com/docs/manual/guide/customtags.html
and it doesn't seem to work on ActionKit's site.  Note that we can't create new tags/filters here -- these are just reimplementations of existing tags/filters that AK has already made :-(

Reply:
Ooooh, I thought we could create new ones. Never mind... I guess I'll just do a JS implementation of this instead.